Casks/k/kodi.rb

```diff
@@ -10,15 +10,14 @@
   desc "Free and open-source media player"
   homepage "https://kodi.tv/"
 
-  # The regex below assumes that the release name will always be one word
-  # (e.g., Leia, Matrix, Nexus, Omega, etc.).
+  # The upstream website is protected by Cloudflare, which prevents
+  # us from fetching the download page. So we check the GitHub repository.
```

Comment:
```suggestion
  # The upstream website is protected by Cloudflare, which prevents us from
  # fetching the download page. The GitHub releases link to Kodi releases, so
  # we check the latest GitHub release as a workaround.
```

This may just be a me thing but I wrap comments at 80 characters (I'm not sure if there's an argument for 70). Besides that, I've expanded the comment to explain why checking GitHub releases is reasonable in this context (i.e., it seems like the GitHub release is created after the website is updated, so we hopefully shouldn't have to worry about whether the dmg file will be available when livecheck surfaces the version).